src/components/SensorCard/index.tsx

```diff
@@ -14,7 +14,7 @@ export interface SensorCardPropType extends ParsedSensorType {
 }
 
 const DISPLAYABLE_RECORDS_AMOUNT = 20;
-const DESCRIPTION_MAX_LENGTH = 150;
```

Comment:
Exporting the same thing twice in AccountCard/index.tsx and SensorCard/index.tsx
Would it make sense to have this in one place or are these lengths bound to visual space that is different on each card?

Reply:
See comment above ⬆️

Reply:
👍🏽